sonar-update-center-mojo/src/main/java/org/sonar/updatecenter/mojo/GenerateHtmlHeadersMojo.java

```diff
@@ -23,34 +23,28 @@
 import org.apache.maven.plugin.AbstractMojo;
 import org.apache.maven.plugin.MojoExecutionException;
 import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
 
-/**
- * @goal generate-html
- * @requiresProject false
- */
+@Mojo(name = "generate-html", threadSafe = true, requiresProject = false)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder annotation arguments to match the order of declaration. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS3340)